packages/public-api/src/web.ts

```diff
@@ -172,10 +172,10 @@ export enum LaunchPage {
 }
 
 export enum DataProvider {
-    COIN_GECKO = 0,
-    COIN_MARKET_CAP = 1,
-    UNISWAP_INFO = 2,
-    NFTSCAN = 3,
+    CoinGecko = 'CoinGecko',
```

Comment:
Caution! Please make sure no settings store any of them.

Reply:
Renamed